www/dashboard.php

```diff
@@ -31,7 +31,7 @@
 require __DIR__ . '/_includes.php';
 
 set_time_limit(180);
-$session = SimpleSAML_Session::getInstance();
```

Comment:
Please read all the changes. I changed many files to the best of my knowledge and checked every files for (re)use of $session. If I removed it, it wasn't used. If I changed it to getSession it was needed later.